MobileDevProject/Room.swift

```diff
@@ -0,0 +1,195 @@
+import Foundation
+import XLActionController
+
+open class RoomCell: ActionCell {
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        initialize()
+    }
+
+    func initialize() {
+        backgroundColor = .clear
+        let backgroundView = UIView()
+        backgroundView.backgroundColor = UIColor.white.withAlphaComponent(0.3)
+        selectedBackgroundView = backgroundView
+        actionTitleLabel?.textColor = .white
+        actionTitleLabel?.textAlignment = .left
+
+    }
+}
+
+public struct RoomHeaderData {
+
+    var title: String
+    var subtitle: String
+//    var image: UIImage
+
+    public init(title: String, subtitle: String) {
+        self.title = title
+        self.subtitle = subtitle
+//        self.image = image
+    }
+}
+
+open class RoomHeaderView: UICollectionReusableView {
+
+    open lazy var imageView: UIImageView = {
+        let imageView = UIImageView(frame: CGRect.zero)
+        imageView.image = UIImage(named: "sp-header-icon")
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    open lazy var title: UILabel = {
```

Comment:
Title works as a variable name but may be clearer if it was "room" or something similar